hoodie-common/src/main/java/com/uber/hoodie/common/util/collection/DiskBasedMap.java

```diff
@@ -215,7 +249,15 @@ public void clear() {
 
   @Override
   public Collection<R> values() {
-    throw new HoodieException("Unsupported Operation Exception");
```

Comment:
This is something that will result in OOMing of the process if allowed to run, I don't think we should support this. I like the valueStream() that you provided below..